### ISSUE TITLE
Increased brain natriuretic peptide (BNP)

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6560]
**Preferred term label:** Brain natriuretic peptide (BNP)
**Synonyms**
Increased ventricular natriuretic peptide
Increased B-type natriuretic peptide

**Definition (free text, please give PubMed ID)**
BNP is a hormone secreted by ventricular cardiomyocytes in response to stretching caused by increased ventricular blood volume. BNP is synthesised as a prohormone (proBNP) comprising 108 amino acids. After the release into the circulation proBNP is cleaved in equimolar amounts into the biologically active BNP (32 amino acid), and the biologically inactive 76 amino acid N‐terminal fragment (NT‐proBNP). BNP has a number of physiological effects including natriuresis/diuresis, peripheral vasodilation, inhibition of renin-angiotensin-aldosterone system (RAAS) and the sympathetic nervous system (SNS). PMID: 16698841

**Parent term (use hpo.jax.org/app)**
Abnormal B-type natriuretic peptide level HP:0031138

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
